src/ignore_path.rs

```diff
@@ -42,14 +42,19 @@ mod test {
         match option_env!("CFG_RELEASE_CHANNEL") {
             // this test requires nightly
             None | Some("nightly") => {
-                let config =
-                    Config::from_toml(r#"ignore = ["foo.rs", "bar_dir/*"]"#, Path::new(""))
-                        .unwrap();
+                let config = Config::from_toml(
+                    r#"ignore = ["foo.rs", "bar_dir/*", "foo\bar"]"#,
```

Comment:
With the following change to the ignore list, I think it would make the assertion L56 stronger: `foo/bar/true.rs` would be a match.

```suggestion
                    r#"ignore = ["foo.rs", "bar_dir/*", "foo\bar\*"]"#,
```

Reply:
Thanks @scampi, good catch! I'll update accordingly and also add another assertion to catch the match and no match scenarios with the windows-style paths

Reply:
Also I assume that the commits will be squashed on merge, but please let me know if you'd prefer I squash the commits myself